eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -431,6 +431,10 @@ jobs:
         displayName: 'Libraries artifacts (AllConfigurations)'
         cleanUnpackFolder: false
 
+  - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
```

Comment:
Merge with the if statement just below?

Reply:
Done in 43a3d87b96323df90d67326f252eb632c56611ab.